setup/install/providers/aws/index.md

```diff
@@ -9,16 +9,28 @@ sidebar:
 
 ### Concepts
 
-There are two types of Accounts in the Spinnaker AWS provider; however, the
-distinction is not made in how they are configured using Halyard, but instead
-how they are configured in AWS.
+There are two types of Accounts in the Spinnaker AWS provider: __AWS Managing__ account and __AWS Managed__ account(s)
 
-1. Managing accounts. There is always exactly one managing account, this
+From the Spinnaker perspective, [Halyard](https://www.spinnaker.io/reference/halyard/) configures Spinnaker to use the __AWS Managing__ account and take control of the __AWS Managed__ account(s).
+
+**_Note_** The AWS IAM structure must be set up prior to adding the Spinnaker AWS Provider with Halyard.
+
+
+From the AWS perspective, __AWS Managing__ account will assume control of the __AWS Managed__ account(s) through the use of AWS IAM Roles. By assuming a role across AWS Accounts is how Spinnaker can control AWS resources from multiple __AWS Managed__ accounts.
+
+Refer to [AWS IAM Providing Access to multiple AWS Accounts](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_common-scenarios_aws-accounts.html) for AWS technical details.
+
+
+1. __AWS Managing__ account. There is always exactly one managing account, this
```

Comment:
Some minor things in this:

- period after `managing account` instead of comma
- replace `assumes roles` with `uses to assume roles`